services/motion/builtin/move_request.go

```diff
@@ -69,14 +66,24 @@ func (mr *moveRequest) execute(ctx context.Context, plan motionplan.Plan) moveRe
 	}
 
 	// the plan has been fully executed so check to see if the GeoPoint we are at is close enough to the goal.
-	errorState, err := mr.kinematicBase.ErrorState(ctx, waypoints, len(waypoints)-1)
+	success, err := mr.deviatedFromPlan(ctx, waypoints, len(waypoints)-1)
```

Comment:
It is confusing to me that a function called `deviatedFromPlan` returning `true, nil` means that the robot succeeded reaching its destination. If we want `true` to mean `success` could I recommend calling this function `reachedPlanDestination`?

Reply:
Also, under what circumstances will it return an error? Could we please document that soemwhere?

Reply:
Renaming to `followingPlan` should make this more clear.  

I don't think its a difficult function to trace the errors in so I don't think that warrants special attention